src/table.ts

```diff
@@ -1238,13 +1249,11 @@ class Table extends common.ServiceObject {
       }
 
       // Read the file into a new write stream.
-      return fs
+      const jobWritable = fs
         .createReadStream(source)
-        .pipe(this.createWriteStream_(metadata))
-        .on('error', callback)
```

Comment:
What will happen now if this stream errors?

Reply:
promise rejection (throw). `pEvent` defaults to listening for `error` event, and rejects the promise.